pages/instructorGradebook/instructorGradebook.ejs

```diff
@@ -73,7 +106,7 @@
                 <td><%= row.role %></td>
                 <% row.scores.forEach(function(score, iScore) { %>
                 <td>
-                  <% if (score.score_perc == null) { %>
+                  <% if (score.score_perc === null) { %>
```

Comment:
The `== null` will catch both `null` and `undefined`, which is why I try and always use it by default. In this case I think we are probably safe with `===` but as a matter of safety I'd tend to prefer `==` unless there is a specific reason we want the triple. (there's another case of this below)

Reply:
Ok, I'll change back in the next commit. For some reason I was thinking that `null` would be equal to 0, but indeed that's not the case.